Coding-Convetion-IntelliJ.xml

```diff
@@ -0,0 +1,37 @@
+<code_scheme name="Project" version="173">
+  <codeStyleSettings language="Groovy">
+    <indentOptions>
+      <option name="USE_TAB_CHARACTER" value="true" />
+    </indentOptions>
+  </codeStyleSettings>
+  <codeStyleSettings language="HTML">
+    <indentOptions>
+      <option name="KEEP_INDENTS_ON_EMPTY_LINES" value="true" />
```

Comment:
why we keep indent on empty line ??

Reply:
This is not so important, we can discuss to find the final decision 😘

Reply:
We should not keep the indents on an empty line. because when we compare 2 versions, some compare tool also compare space or empty character as well, make us harder to resolve conflict(if has), remove all indent on empty line keep us out of that problem.

Reply:
Agrre with you, already changed